cdk/cdk.json

```diff
@@ -1,5 +1,5 @@
 {
-	"app": "tsx bin/cdk.ts",
+	"app": "ts-node-dev bin/cdk.ts",
```

Comment:
we discussed off-thread that the reason for using `ts-node-dev` here is because `tsx` restarts if the user presses Enter, which doesn't work with the current CLI setup